config/schema/oe_multilingual.schema.yml

```diff
@@ -0,0 +1,10 @@
+oe_multilingual.settings:
+  type: mapping
+  label: 'Language category'
+  mapping:
+    category:
+      type: string
+      label: 'The category of the language.'
+
+language.entity.*.third_party.oe_multilingual:
+  type: oe_multilingual.settings
```

Comment:
Is it necessary to declare custom type? If there are no plans to reuse, it is ok to declare structure here.